pkg/jx/cmd/cmd.go

```diff
@@ -21,14 +21,18 @@ const (
     `
 )
 
+var _factory Factory
```

Comment:
I'm not sure if this is the best way to pass the `factory` object over to the `preRun` function, but I couldn't find an easy way to do it without making it a variable at the package level.